packages/block-editor/src/hooks/use-editor-wrapper-styles.native.scss

```diff
@@ -0,0 +1,31 @@
+.block-editor-hooks__use-editor-wrapper-styles {
```

Comment:
I think we could drop `block-editor-hooks` from the selector names. I'm not sure the context it adds is all that meaningful and it significantly elongates the selectors. WDYT?

Reply:
I agree, it does make the names to be too long. I'll make the change 👍

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/49616/commits/696d5a97dabea9f96168ccdf80af6b2c8491df3b#diff-5fe52ed070a8b8421565978bb3b671eb1b0d89d746db2b63a7203b52c937e0a7